ffv1.md

```diff
@@ -401,49 +402,49 @@ Early experimental versions of FFV1 used the CABAC Arithmetic coder from H.264 a
 
 #### Range Binary Values
 
-To encode binary digits efficiently a Range coder is used. `C(i)` is the i-th Context. `B(i)` is the i-th byte of the bytestream. `b(i)` is the i-th Range coded binary value, `S(0,i)` is the i-th initial state. The length of the bytestream encoding n binary symbols is `j(n)` bytes.
+To encode binary digits efficiently a Range coder is used. `C_{i}` is the i-th Context. `B_{i}` is the i-th byte of the bytestream. `b_{i}` is the i-th Range coded binary value, `S_{0, i}` is the i-th initial state. The length of the bytestream encoding n binary symbols is `j_{n}` bytes.
```

Comment:
The ascii art output looks good, but in the HTML output with SVG, this sentence somewhat contradicts the SVG, but that's true before and after this PR.

Reply:
> The ascii art output looks good, but in the HTML output with SVG, this sentence somewhat contradicts the SVG, but that's true before and after this PR.

I have put this commit to the end of the queue for this PR, and merged it with the first from @dericed in https://github.com/FFmpeg/FFV1/pull/225 (which could be closed, I also added the 2 last commits from the linked PR to this PR as it is related).

@dericed @michaelni please review the changes.